pkg/log/fiano_logger.go

```diff
@@ -0,0 +1,32 @@
+package log
+
+import (
+	"github.com/facebookincubator/go-belt/tool/logger"
+	fianoLog "github.com/linuxboot/fiano/pkg/log"
+)
+
+var _ fianoLog.Logger = FianoLogger{}
+
+// FianoLogger is just a placeholder for a logger, which does nothing
```

Comment:
This isn't true? We've gone from it doing nothing to being a wrapper around a logger. Also should the methods below not be checking if Backend is nil?

Reply:
Yeah, replaced `DummyLogger` with `FianoLogger`, but forgot to update the comments. Fixed.

Reply:
What about the nil checks? You do `fianoLog.DefaultLogger = log.FianoLogger{}` in various places, which I think means if Fiano tries to log it's going to try to call a function on Backend which will be nil?

Reply:
IDE replaced it everywhere (I used this function) and I just didn't know about those cases :(

Thanks for noticing! Fixed.